src/lsm/grid.zig

```diff
@@ -182,6 +190,10 @@ pub fn GridType(comptime Storage: type) type {
             const cache_blocks = try allocator.alloc(BlockPtr, options.cache_blocks_count);
             errdefer allocator.free(cache_blocks);
 
+            var cache_valid =
+                try std.DynamicBitSetUnmanaged.initEmpty(allocator, options.cache_blocks_count);
```

Comment:
To double-check my understanding of memory usage here:

If the user allocates 32 GiB of grid cache at runtime, with 64 KiB blocks, then we expect this allocation to require:

```
32*1024*1024*1024 / 65536 / 8 = 65536 bytes
```